src/sass/components/_buttons.scss

```diff
@@ -280,16 +280,16 @@
 
 .rvt-button--loading,
 .rvt-button--loading[disabled] {
-  background-color: $color-blue--100;
-  border-color: $color-blue--200;
-  color: $color-blue--600;
+  background-color: $color-black--100;
+  border-color: $color-black--400;
+  color: $color-black--700;
   justify-content: center;
   position: relative;
 
   &:hover {
-    background-color: $color-blue--100;
-    border-color: $color-blue--200;
-    color: $color-blue--600;
+    background-color: $color-black--100;
+    border-color: $color-black--400;
+    color: $color-black--700;
   }
 
```

Comment:
Would need to add styles to change the border color of the `.rvt-loader` while in the `--loading`/`disabled` state:

```sass
border-top-color: $color-black--400;
border-right-color: $color-black--400;
border-bottom-color: $color-black--400;
```